kiwi/boot/image/dracut.py

```diff
@@ -110,6 +111,11 @@ def create_initrd(self, mbrid=None, basename=None, install_initrd=False):
                     kernel_details.version
                 ]
             )
+            process = CommandProcess(
+                command=cmd_call, log_topic='initrd creation',
+                stderr_to_stdout=True
+            )
+            process.poll()
```

Comment:
Hmm, I don't understand why we poll on the process without any interactive status information. That should not be necessary and could be done simpler imho:

```python
dracut_call = Command.run(...)
log.debug(dracut_call.output)
# or maybe: log.debug(dracut_call.error) if dracut writes to stderr
```

Any reason why it can't be done that way ?

Reply:
I like the stderr to stdout extension though

Reply:
Well yes I agree there is no need to get this output using poll. I just did it this way because it is more pleasant if you are actively looking at the logs during the build. Dracut output is reasonably huge and wanted to avoid dumping all output at once after the process is done. Dracut output is produced and shown to the user sequentially like a `zypper in -y pkg` call. So I thought about printing each line as soon as dracut produces it.

I know this is a weak argument :sweat_smile: I am opened to change that if you prefer.

Reply:
ok got it, makes sense to me. But can you add a short line of explanation. I will surely forgot it and I bed someone will think this needs to be refactored :) Thanks